addons/mail/models/mail_mail.py

```diff
@@ -3,12 +3,10 @@
 import base64
 import datetime
 import logging
+import psycopg2
```

Comment:
Yep. It has not be done while migrating the module to the new API, so maybe I can do it now.